proto/zitadel/resources/action/v3alpha/action_service.proto

```diff
@@ -294,34 +395,43 @@ service ZITADELActions {
 }
 
 message CreateTargetRequest {
-  Target target = 1;
+  optional zitadel.object.v3alpha.Instance instance = 1;
+  Target target = 2 [
```

Comment:
The object doesn't have any validations is that correct?

Reply:
I'll check it

Reply:
I added validations